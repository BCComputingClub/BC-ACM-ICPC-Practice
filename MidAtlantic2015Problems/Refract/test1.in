600 600 1000 1.333 1.01
600 1200 4000 1.5 1.01
1 100 10000 2.5 1.01
50 100 10000 2.5 1.01
100 100 10000 2.5 1.01
200 100 10000 2.5 1.01
300 100 10000 2.5 1.01
400 100 10000 2.5 1.01
600 100 10000 2.5 1.01
700 100 10000 2.4 1.01
800 100 10000 2.3 1.01
400 100 9000 2.2 1.01
400 100 8000 2.1 1.01
400 100 7000 2.0 1.01
400 100 6000 1.9 1.01
400 100 5000 1.8 1.01
400 100 3000 2.0 1.01
400 100 2000 2.0 1.01
400 100 1000 2.0 1.01
400 100 100 2.0 1.01
400 100 100 2.0 1.01
500 100 10 2.0 1.01
400 100 0 2.2 1.01
800 200 100 2.3 1.01
800 300 500 2.4 1.01
800 500 2000 2.5 1.01
700 700 3000 2.5 1.01
600 800 5000 2.5 1.01
500 1000 6000 2.5 1.01
400 3000 8000 2.5 1.01
400 10000 10000 2.5 1.01
400 5000 5000 2.0 1.01
400 5000 5000 1.5 1.01
400 5000 5000 1.1 1.01
100 2000 8000 2.5 1.1
100 2000 10000 2.5 1.2
100 2000 10000 2.5 1.5
100 2000 10000 2.5 1.75
100 2000 10000 2.5 2.0
100 2000 8000 2.5 2.25
500 100 10000 2.25 1.01
500 100 10000 2.0 1.01
500 100 10000 1.5 1.01
500 100 10000 1.25 1.01
500 100 9000 1.02 1.01
500 100 2000 2.25 1.01
500 100 1000 1.02 1.01
0 0 0 0 0
