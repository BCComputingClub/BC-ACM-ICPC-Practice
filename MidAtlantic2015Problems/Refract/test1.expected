44.37
11.51
0.01
0.29
0.58
1.15
1.73
2.30
3.45
4.02
4.60
2.56
2.88
3.30
3.85
4.64
7.74
11.63
22.89
77.46
77.46
88.96
90.00
83.57
61.29
23.95
14.52
7.35
5.14
3.43
4.09
10.58
21.85
42.73
0.82
0.64
0.67
0.71
0.78
1.48
2.88
2.88
2.89
2.90
3.43
14.37
30.81
